.github/workflows/try-pr-on-eclipse-che-hosted-by-red-hat.yaml

```diff
@@ -0,0 +1,15 @@
+on: pull_request_target
+
+jobs:
+  example_comment_pr:
+    runs-on: ubuntu-latest
+    name: Try the PR on Eclipse Che hosted by Red Hat
+    steps:
+      - name: Checkout
```

Comment:
do we need to checkout the project ?

Reply:
looks like yes according to the usage - https://github.com/ibuziuk/actions-comment-pull-request#usage

Reply:
Indeed, looks like the checkout step is not needed

Reply:
Removed